src/sass/style.scss

```diff
@@ -1,4 +1,5 @@
 @import "foundation/functions";
+@import "foundation";
```

Comment:
`foundation.css` imports **all** foundation components. We don't want to do this. Import only top-bar component and move it below.